cvat/settings/base.py

```diff
@@ -787,3 +787,5 @@ class CVAT_QUEUES(Enum):
             "cron_string": "0 8 * * *",
         }
     )
+
+USER_LAST_ACTIVITY_UPDATEP_PERIOD = timedelta(days=1)
```

Comment:
```suggestion
USER_LAST_ACTIVITY_UPDATE_PERIOD = timedelta(days=1)
```

Also, I'm doubtful about using "period" here, because we don't update it _periodically_. Maybe something like `MIN_INTERVAL` would be better?